business/checkers/workloads/ambient/ambient_workload_checker.go

```diff
@@ -0,0 +1,183 @@
+package workloads
+
+import (
+	security_v1 "istio.io/client-go/pkg/apis/security/v1"
+
+	"github.com/kiali/kiali/config"
+	"github.com/kiali/kiali/models"
+)
+
+type AmbientWorkloadChecker struct {
+	Cluster               string
+	Conf                  *config.Config
+	Workload              *models.Workload
+	Namespace             string
+	Namespaces            models.Namespaces
+	AuthorizationPolicies []*security_v1.AuthorizationPolicy
+}
```

Comment:
I'd suggest adding a constructor for this checker, like `NewAmbientWorkloadChecker(...argPerAttribute)`. It's a safer pattern because it protects against not initializing a Struct field. We've been adding these when touching existing code, and since this is a new checker, it would be good to start with a constructor.  You can see here how we've introduced several for the business services:

https://github.com/kiali/kiali/blob/master/business/layer.go#L50-L65

No need to worry about existing checkers, I'm only suggesting we add one here because it's a new checker.